parse.y

```diff
@@ -49,6 +49,7 @@ typedef struct {
 } yystype;
 #define YYSTYPE yystype
 
+const char *yyfn;
```

Comment:
Is there a common header where this could be defined? Having multiple non type checked declarations of a given variable is always weird :/.

Reply:
I was just copying pre-existing style.  We can (and should) refactor this a bit.

Reply:
Refactored in 3566fc5

Reply:
I rather not refactor such things as this just adds more diffs and makes merging upstream commits harder without any real benefit.

Reply:
I backed out the refactoring.